traits/src/types.rs

```diff
@@ -287,10 +287,10 @@ impl core::fmt::Display for Ciphersuite {
     }
 }
 
-impl From<&Ciphersuite> for u16 {
+impl From<Ciphersuite> for u16 {
```

Comment:
We should remove all instances of `&Ciphersuite` as `Ciphersuite`s are `Copy`able. If we agree on this I can already open an issue for some later cleanup.